tests/test_core.py

```diff
@@ -2630,6 +2640,7 @@ def test_dlfcn_varargs(self):
     self.do_run(src, '100\n200\n13\n42\n',
                 post_build=self.dlfcn_post_build)
 
+  @no_wasm # TODO: this needs to add JS functions to a wasm Table, need to figure that out
```

Comment:
This one is needed as it won't even pass in asm2wasm, so no wasm mode works for it.